app/controllers/branch_crud_controller.rb

```diff
@@ -0,0 +1,40 @@
+class BranchCrudController < ApplicationController
```

Comment:
It's not necessary to include the word `Crud`

Usually, controllers are in plural as well

I'd suggest something like `BranchesController`, but it's not necessary, it was just a code smell to me 😄

Reply:
We changed the name